src/v1/cluster_manager_client_config.json

```diff
@@ -36,143 +36,142 @@
           "retry_params_name": "default"
         },
         "UpdateCluster": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "UpdateNodePool": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetNodePoolAutoscaling": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetLoggingService": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetMonitoringService": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetAddonsConfig": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetLocations": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "UpdateMaster": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "SetMasterAuth": {
-          "timeout_millis": 60000,
+          "timeout_millis": 45000,
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
         "DeleteCluster": {
-          "timeout_millis": 60000,
-          "retry_codes_name": "non_idempotent",
+          "timeout_millis": 20000,
+          "retry_codes_name": "idempotent",
           "retry_params_name": "default"
         },
         "ListOperations": {
-          "timeout_millis": 60000,
+          "timeout_millis": 20000,
```

Comment:
Please double check that this timeout change is expected (the gRPC service config might be incorrect). If it is incorrect, we'll need to fix it separately.